Resources/Prototypes/Radium/Entities/Clothing/OuterClothing/hardsuits.yml

```
@@ -65,3 +65,59 @@
+        Heat: 0.8
+  # phase cloak
+  - type: ToggleClothing
+    action: ActionTogglePhaseCloak
+  - type: ComponentToggler
+    parent: true
+    components:
+    - type: Stealth
+      minVisibility: 0.1
+      lastVisibility: 0.1
+  - type: PowerCellDraw
+    drawRate: 0 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)